app/models/base_regulation.rb

```diff
@@ -7,13 +7,32 @@ class BaseRegulation < Sequel::Model
   plugin :time_machine, period_start_column: :base_regulations__validity_start_date,
                         period_end_column: :effective_end_date
   plugin :conformance_validator
+  plugin :dirty
 
   set_primary_key [:base_regulation_id, :base_regulation_role]
 
   include ::FormApiHelpers::RegulationSearch
 
-  one_to_one :complete_abrogation_regulation, key: [:complete_abrogation_regulation_id,
-                                                    :complete_abrogation_regulation_role]
+  one_to_one :complete_abrogation_regulation,
```

Comment:
these will need to be pulled across to tariff backend @alexey-klimuk

Reply:
@alexey-klimuk copy-pasting

Reply:
Bascially anything that is updating the model associations / code

Reply:
👌